src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5074,4 +5071,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_VoidInTuple" xml:space="preserve">
     <value>A tuple may not contain a value of type 'void'.</value>
   </data>
+  <data name="ERR_NonTaskMainCantBeAsync" xml:space="preserve">
+    <value>A void or int returning Main cannot be async</value>
```

Comment:
>Main [](start = 35, length = 4)

It is probably better to use term "entry point" instead of "Main". It was used by the original message and is more precise because, in general, void or int returning methods named Main can be async.